docs/xml/elements/electoral_district.rst

```diff
@@ -1,24 +1,56 @@
 ElectoralDistrict
 =================
 
-.. todo::
+The ``ElectoralDistrict`` object represents the geographic area in which a single contest is held.
```

Comment:
Instead of "a single contest is" I'd say "contests are"; e.g., there are multiple statewide contests.